src/test/ui/pattern/usefulness/doc-hidden-fields.stderr

```diff
@@ -0,0 +1,59 @@
+error: pattern requires `..` due to inaccessible fields
+  --> $DIR/doc-hidden-fields.rs:15:9
+   |
+LL |     let HiddenStruct { one, two, } = HiddenStruct::default();
+   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^
+   |
+help: ignore the inaccessible and unused fields
+   |
+LL |     let HiddenStruct { one, two, .., } = HiddenStruct::default();
```

Comment:
Why is there a comma after this `..` but not after the one on line 20?

Reply:
It seems that if the source text has the trailing comma i.e. `let Struct { a, b, } = thing;` then the last item, whatever it is (field name or rest operator), has the trailing comma. I fixed it by removing it from the example.